packages/ui/src/components/Checkbox/checkbox.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Checkbox as DefaultCheckbox, CheckboxProps as DefaultCheckboxProps } from 'antd'
+import { Typography } from '../Typography'
+
+const CHECKBOX_CLASS_PREFIX = 'condo-checkbox'
+
+type CondoCheckboxProps = {
+    children?: string
```

Comment:
maybe better name would be `label`?
```suggestion
    label?: string
```